rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -180,12 +178,7 @@ def constant_value_to_py(type_, value):
         if type_.typename == 'bool':
             return 'True' if value else 'False'
 
-        if type_.typename in (
-            'int8', 'uint8',
-            'int16', 'uint16',
-            'int32', 'uint32',
-            'int64', 'uint64',
-        ):
+        if type_.typename in INTEGER_TYPES:
             return str(value)
 
         if type_.typename == 'char':
```

Comment:
@dirk-thomas same about an equivalent `CHAR_TYPE` (I know there's none, I just find it awkward to use the variable in some places and the raw string in others).

Reply:
See https://github.com/ros2/rosidl_python/pull/45#discussion_r277388399. Why should we create a constant for that name (`CHAR_TYPE = 'char'`)? What would be the advantage?

Reply:
Just copying a comment from above which applies here:

> The linter plugins can catch misspellings of `BOOLEAN_TYPE`, but not misspellings of `'bool'`.